databases/week1/Week1.sql

```diff
@@ -0,0 +1,56 @@
+-- 1.Find out how many tasks are in the task table
+SELECT  count(*)
+ FROM task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT count(*)
+FROM task
+WHERE due_date  is NULL;
+
+-- 3.Find all the tasks that are marked as done
+SELECT count(*)
+FROM task
+WHERE status_id = 3;
+
+-- 4.Find all the tasks that are not marked as done
+SELECT count(*)
+FROM task
+WHERE status_id != 3;
+
+-- 5.Get all the tasks, sorted with the most recently created first
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+-- 6.Get the single most recently created task
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+-- 7.Get the title and due date of all tasks where the title or description contains database
+SELECT title,due_date 
+FROM task
+WHERE title REGEXP "database"
+OR description REGEXP "database";
+
+-- 8.Get the title and status (as text) of all tasks
+SELECT t.title, s.name  
+FROM task t
+JOIN  status s
+on t.status_id = s.id;
```

Comment:
Great! :+1: And nice that you are using aliases for the table names. It can be really useful when dealing with long table names